### ISSUE TITLE
kommentarsymbol nicht da

### ISSUE THREAD

**Issue by stop5** [eknoes/golem-android-reader#36]
Bei sehr neuen Artikeln ist es nicht möglich einen Kommentar abzugeben. das dazugehörige Symbol ist nicht da und der teil des Textes wo der Link normalerweise ist, ist abgeschnitten.
![Screenshot_20210727-160926_Golem.de_1.png](https://user-images.githubusercontent.com/6901125/127169617-a6980d94-b876-4cee-b8a6-8dc870bfe6f6.png)

**Comment by eknoes** [comment]
Manchmal scheint der nur bei dem neuesten Artikel zu fehlen, kannst du das bestätigen?

**Comment by stop5** [comment]
Richtig. Die zeit bis es erscheint ist allerdings unterschiedlich, es ist ebenfalls unabhängig davon ob es kommentare gibt oder nicht.
